src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -112,6 +112,24 @@ class VolumeBar extends Slider {
     this.el_.setAttribute('aria-valuetext', volume + '%');
   }
 
+  /**
+   * When user starts dragging the VolumeBar, store the volume and listen for
+   * the end of the drag. When the drag ends, if the volume was set to zero,
+   * set lastVolume to the stored volume.
+   *
+   * @listens slideractive
+   */
+  updateLastVolume() {
+    const volumeBeforeDrag = this.player_.volume();
+
+    this.on('sliderinactive', function setLastVolume() {
```

Comment:
You can actually use `this.one('sliderinactive'` here that way you don't have to use `this.off` below. I would also say that you can just use an arrow function here rather than a named one. ex: 
```js
this.one('sliderinactive', () => {
// current function goes here
});
```